packages/react-text/src/components/Headline/Headline.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from 'react';
+import { makeStyles, mergeClasses } from '@fluentui/react-make-styles';
+import { HeadlineProps } from './Headline.types';
+import { renderText, useText, useTextStyles } from '../Text/index';
+import { typographyStyles } from '../../index';
+
+/**
+ * Styles for the root slot
+ */
+const useStyles = makeStyles({
+  root: typographyStyles.headline,
+});
+
+/**
+ * Text wrapper component for the Headline typography variant
+ */
+export const Headline = React.forwardRef<HTMLElement, HeadlineProps>((props, ref) => {
+  const styles = useStyles();
```

Comment:
hmm so we won't specify semantic tags for particular typography types ? should that be handled by consumer via `as` prop ?

Reply:
We had considered this and decided to leave it as the user's responsibility to decide on the role and semantics that the text will have for their use cases. Not for all users it would make sense to have Display as `h1` or maybe Title1 as `h2` and so on.
Accessibility wise, it is better to keep the neutral approach where everything is a span if the user does not specify differently, because if we force semantics and the user does not care about a11y it will result in unwanted usability for screen readers.